src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/ServiceController.php

```diff
@@ -80,6 +80,25 @@ public function __construct(
         $this->serviceStatusService = $serviceStatusService;
     }
 
+    /**
+     * @Method({"GET"})
+     * @Route("/", name="service_overview")
+     * @Template()
+     */
+    public function overviewAction()
+    {
+        $serviceOptions = $this->authorizationService->getAllowedServiceNamesById();
+        $services = $this->serviceService->getServicesByCurrentServiceIds($serviceOptions);
```

Comment:
Isn't the array returned from the `getAllowedServiceNamesById` not an array with names as suggested with the method name? Maybe it's better to use a valueobject or collection to return the required services.

Reply:
See ca4ed06

Reply:
Yes this is a just comment! Let me improve this.